### PR TITLE
Add bridges-2

### PR DESCRIPTION
**Pull request by abillscmu**
I also use bridges 2 but am not an admin there, so wanted to separate this one too.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -172,12 +173,15 @@ installation and/or support for using Julia to its users:
...
+[Pittsburgh Supercomputing Center](https://www.psc.edu) | [Bridges-2](https://www.psc.edu/resources/bridges-2/) | ✅ | ✅  | ✅ | [AMD EPYC, Intel Xeon](https://www.psc.edu/resources/bridges-2/) | [Nvidia V100](https://www.psc.edu/resources/bridges-2/) | [1](https://www.psc.edu/resources/software/julia/)
 [ULHPC, U Luxembourg](https://hpc.uni.lu/) | [Aion](https://hpc-docs.uni.lu/systems/aion/), [Iris](https://hpc-docs.uni.lu/systems/iris/) | ✅ | ? | ✅ | [AMD EPYC Rome](https://hpc-docs.uni.lu/systems/aion/) + [Intel Xeon Broadwell/Skylake](https://hpc-docs.uni.lu/systems/iris/) | [Nvidia Tesla V100](https://hpc-docs.uni.lu/systems/iris/) | [1](https://hpc-docs.uni.lu/software/maths/julia/)
 [ZDV, U Mainz](https://hpc-en.uni-mainz.de/) | [MOGON II](https://hpc-en.uni-mainz.de/high-performance-computing/systeme/) | ✅ | ? | ? | [Intel Xeon Broadwell + Skylake](https://hpc-en.uni-mainz.de/high-performance-computing/systeme/) | no | [1](https://mogonwiki.zdv.uni-mainz.de/dokuwiki/start:development:scripting_languages:julia)
 
 
+
```
Comment (sloede) [inline README.md:184]:
```suggestion
```
unnecessary blank line

File: README.md [hunk 2]
```diff
@@ -221,6 +224,7 @@ PRs or via private communication:
 * Pedro Ojeda ([@pojeda](https://github.com/pojeda))
 * Samuel Omlin ([@omlins](https://github.com/omlins))
 * Dinindu Senanayake ([@DininduSenanayake](https://github.com/DininduSenanayake))
+* Alexander Bills ([@abillscmu](https://github.com/abillscmu))
```
Comment (sloede) [inline README.md:227]: Would you mind putting your name alphabetically in the list as well?

File: README.md [hunk 3]
```diff
@@ -172,12 +173,15 @@ installation and/or support for using Julia to its users:
...
 [NeSI](https://www.nesi.org.nz/) | [Mahuika, Māui](https://www.nesi.org.nz/services/high-performance-computing-and-analytics/platforms) | ✅ | ✅ | ✅ | [Intel Xeon Broadwell/Cascade Lake + AMD EPYC Milan](https://www.nesi.org.nz/services/high-performance-computing-and-analytics/platforms) | [Nvidia Tesla P100, A100](https://support.nesi.org.nz/hc/en-gb/articles/360001471955) | [1](https://support.nesi.org.nz/hc/en-gb/articles/360001175895-Julia)
+[Open Science Grid](https://opensciencegrid.org) | N/A | x | ✅  | ?  | [Various](https://support.opensciencegrid.org/support/solutions/articles/5000632058-computation-on-the-open-science-pool) | [Various](https://support.opensciencegrid.org/support/solutions/articles/5000653025-using-gpus-on-the-ospool) | [1](https://support.opensciencegrid.org/support/solutions/articles/12000078187-using-julia-on-the-ospool)
```
Comment (sloede) [inline README.md:176]:
```suggestion
[Open Science Grid](https://opensciencegrid.org) | N/A | ❌ | ✅  | ?  | [Various](https://support.opensciencegrid.org/support/solutions/articles/5000632058-computation-on-the-open-science-pool) | [Various](https://support.opensciencegrid.org/support/solutions/articles/5000653025-using-gpus-on-the-ospool) | [1](https://support.opensciencegrid.org/support/solutions/articles/12000078187-using-julia-on-the-ospool)
```
minor style suggestion

**Review by sloede — CHANGES_REQUESTED** [review]
Thanks a lot for putting in the effort to add these new systems to the list!

The PR looks almost ready to be merged. It would be great if you could put your name in the list alphabetically as well.

File: README.md (hunk 2 above)
Comment (abillscmu) [reply README.md]: I have now put them all in alphabetical order

**Review by sloede — APPROVED** [review]
Thanks again for the contributions and for bearing with my sorting fetish 😊

**Pull request merged**